cli/src/commands/create/comments.rs

```diff
@@ -280,7 +280,10 @@ fn upload_batch(
                     labelling.as_deref(),
                     entities.as_ref(),
                 )
-                .context("Could not update labelling for comment")?;
+                .context(format!(
+                    "Could not update labelling for comment `{}`",
+                    comment_uid.0,
+                ))?;
```

Comment:
You probably want `with_context`, which is lazily evaluated - otherwise you're allocating a new string even if the error case isn't hit.

https://docs.rs/anyhow/1.0.45/anyhow/trait.Context.html#tymethod.with_context

```suggestion
                .with_context(|| {
                    format!("Could not update labelling for comment `{}`", comment_uid.0,)
                })?;
```